Description: Disable automatic tests, that are know to be unstable
Author: Martin <debacle@debian.org>
Origin: vendor
Bug: https://github.com/giampaolo/pyftpdlib/issues/420
Bug-Debian: https://bugs.debian.org/856335
Last-Update: 2022-11-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/pyftpdlib/test/runner.py
+++ b/pyftpdlib/test/runner.py
@@ -130,9 +130,10 @@
 class TestLoader:
 
     testdir = HERE
-    skip_files = []
+    skip_files = os.environ.get("EXCLUDETESTS", "").split()
 
     def _get_testmods(self):
+        excludefiles = os.environ.get("EXCLUDETESTS", "").split()
         return [os.path.join(self.testdir, x)
                 for x in os.listdir(self.testdir)
                 if x.startswith('test_') and x.endswith('.py') and
